src/main/java/org/jenkins/tools/test/model/hook/PluginCompatTesterHook.java

```diff
@@ -31,7 +32,8 @@ default boolean check(Map<String, Object> info) {
      *
      * <p>Certain implementations could throw exceptions.
      */
-    Map<String, Object> action(Map<String, Object> moreInfo) throws Exception;
+    Map<String, Object> action(Map<String, Object> moreInfo)
+            throws PluginCompatibilityTesterException;
```

Comment:
Note that hooks can now only throw one of the subclasses of `PluginCompatibilityTester` exception rather than any exception.